bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -133,7 +135,6 @@ private func dummy() {
     _ = NSLocalizedString("Edit cover media", comment: "")
     _ = NSLocalizedString("Edit file", comment: "")
     _ = NSLocalizedString("Edit focal point", comment: "")
-    _ = NSLocalizedString("EDIT LINK", comment: "")
```

Comment:
- Removal
- No code reference

This string is not being used anywhere so we can safely remove it, probably the string changed.